routes/error-tracker.js

```diff
@@ -49,6 +49,47 @@ function ignoreMessageOrException(message, exception) {
 }
 
 /**
+ * @param {string} exception
+ * @return {string} standardized exception
+ */
+function stackTraceConversion(exception) {
+  let chromeStackTraceRegex = /\s*at (?:([^\n]*) )?([^\n]+):(\d+):(\d+)[^]/gm;
```

Comment:
Unfortunately, we're going to have to make this regex a bit more complex to handle cases:

```
    at test (https://google.com/file.js:2:11)
    at (https://google.com/file.js:2:11)
    at test https://google.com/file.js:2:11
    at https://google.com/file.js:1:1
```

The quoted paths should not allow an unmatched `(` or `)` to be present. Something like:

```js
const location = '([^ \\n]+):(\\d+):(\\d+)';
const regex = new RegExp(`^\\s*at (.+ )?(?:${location}|\(${location}\))$`, 'gm');